utils/utils.ts

```diff
@@ -0,0 +1,15 @@
+import {Keyboard, KeyboardEventListener} from 'react-native';
+
+export const keyboardListeners = (
```

Comment:
This should be a hook. `useKeyboardListeners` and it should have the `useEffect` already inside, then you can call it in your components like so: `useKeyboardListeners(func1, func2)` or to make it even more readable, you can do this: `useKeyboardListeners({onShow: func1, onHide: func2})`

Reply:
This way you avoid duplication, by forcing the usage to always render `useEffect` in the component.